community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/ListIndexesProcedure.java

```diff
@@ -46,6 +49,7 @@ protected ListIndexesProcedure(ProcedureName procedureName)
         super( procedureSignature( procedureName )
                 .out( "description", Neo4jTypes.NTString )
                 .out( "state", Neo4jTypes.NTString )
+                .out( "unique", Neo4jTypes.NTBoolean )
```

Comment:
Personally I don't like aggregating all information in one big string because it limits what the client can do with the return value (without resorting to string parsing). I still would prefer returning labels and properties separately, but that is maybe orthogonal. Adding `UNIQUE` changes the way we present them, currently the output is 
```
INDEX ON :User(name) (for uniqueness constraint)
```